openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners.html

```diff
@@ -0,0 +1,30 @@
+{% load i18n %}
+
+<div class="staff__grade__control ui-toggle-visibility is--collapsed">
+    <header class="staff__grade__header ui-toggle-visibility__control">
+        <h3 class="staff__grade__title">
+            <span class="wrapper--copy">
+                <button class="staff__grade__show-form">{% trans "Staff Assessment" %}</button>
+            </span>
+        </h3>
+
+        {% block title %}
+        <span class="staff__grade__status">
```

Comment:
Note that in #825, the counts move into their own template so they can be updated without re-rending the entire staff area.